build.gradle.kts

```diff
@@ -60,6 +60,22 @@ kotlin {
         binaries.executable()
     }
 
+    wasmJs {
+        binaries.library()
+        browser {
+            testTask {
+                // TODO: enable once the tests work with Kotlin/Wasm.
+                enabled = false
```

Comment:
The tests fail with a hard-to-understand error:

```
RuntimeError: array element access out of bounds
```

More:
* https://github.com/krzema12/kaml/actions/runs/9437748515/job/25993965918?pr=3
* thread: https://kotlinlang.slack.com/archives/CT0G9SD7Z/p1717949613990609

I figured that given we have the tests for JS disabled, we can start with disabled for WASM as well. It's not ideal, we should finally revisit missing tests for JS as well.

Reply:
I'd prefer to understand exactly why the tests are failing first - if they're failing because there's something wrong, then I'd like to know that before we publish this

Reply:
Sure, see https://github.com/charleskorn/kaml/pull/567/commits/d1a671cc3d30cb53facd150f54139a7cf84559b4
It's my attempt to make the test pass. Unfortunately, the test task itself still fails on some issue unrelated to the tests themselves, but rather something related to test report creation (`Could not write XML test results for ...`).

Regarding the tests themselves, I found two issues with Wasm:
* [Wasm: float from variable is printed with many decimal points](https://youtrack.jetbrains.com/issue/KT-68948/)
  * IMO a plain bug, but we can try releasing with it because Wasm support can still bring some value if someone doesn't use floating point numbers
* [WASM: Can't catch index out of bounds and divide by 0 exceptions?](https://youtrack.jetbrains.com/issue/KT-59081/)
  * enabling the compiler flag does the trick, but not sure what impact on performance it has (the whole point of the opt-in is performance)

Reply:
Now we either:
* wait until at least the [reporting problem](https://youtrack.jetbrains.com/issue/KT-68950/) is fixed so that the test task succeeds,
* or disable the tests like for JS, release kaml with Wasm support, gather feedback, and in the meantime wait until the issue is fixed.

@charleskorn up to you! If you decide to go with path #2, it's enough to change

```kotlin
    wasmJs {
        binaries.library()
        browser()
        nodejs()
    }
```

into

```kotlin
    wasmJs {
        binaries.library()
        browser {
            testTask {
                // TODO: enable once the tests work with Kotlin/Wasm.
                // See https://youtrack.jetbrains.com/issue/KT-68950/
                enabled = false
            }
        }
        nodejs {
            testTask {
                // TODO: enable once the tests work with Kotlin/Wasm.
                // See https://youtrack.jetbrains.com/issue/KT-68950/
                enabled = false
            }
        }
    }
```

Reply:
@charleskorn I checked in my project by copying kaml there and enabling Wasm support: https://github.com/typesafegithub/github-actions-typing-editor/tree/main/kaml
So it works! That's why I think disabling the tests until the underlying issues are solved and releasing without the tests enabled beings some value. Let me know about your decision!

Reply:
I've disabled the tests to make the PR ready to merge.